docs/semgrep-cloud-platform/teams.md

```diff
@@ -0,0 +1,290 @@
+
+This document walks you through the following:
+
+- How to approach team management and project access in Semgrep
+- How to create, view, update, and delete teams and subteams
+- How to assign or unassign projects to teams
+
+## Roles and visibility
+
+A user is any person who has been added to your organization in Semgrep. Users can be administrators (admins), managers, or members.
+
+<dl>
+<dt>Admin</dt>
+<ul><li>A user who has access to all features, resources, and projects of their Semgrep deployment. Admins can also change the role of members and managers.</li>
+<li>By definition, an admin does not need to be assigned to a team to have access to the resources of a project. As such, an admin can't be removed from a team.</li>
+<li>An org admin can change the role of any other admin or user, including a fellow admin.</li></ul>
+<dt>Manager</dt>
+<ul>
+<li>A user who can grant access to projects by creating subteams and assigning members to these subteams.</li>
+<li>A manager role is restricted to the teams where they have been assigned as a manager. Users can be managers of some projects, but members for others. For more information, see the <a href="#the-manager-role">manager role</a>.</li>
+</ul>
+<dt>Member</dt>
+<ul>
+<li>A user who has access to some features, resources, and projects of their Semgrep deployment.</li>
+<li>By default, a member starts out with no access to any project.</li>
+<li>To grant members access to a project and its findings, you must add the members to a team, and that team must be assigned to the project.</li>
+<li>Members can scan their local or personal repositories through a personal account.</li>
+</ul>
+</dl>
+
+![A member's view of the Projects page.](/img/access-member-view.png)
+**Figure**. A member's view of the Projects page. It displays projects that are assigned to the team they are a member of, but they cannot edit a project nor can they scan new projects in their organizational account.
+
+### Page visibility per role
+
+| Page      | Member     | Manager    | Admin | Notes |
+| ----      | ------     | -------    | ----- | ----- |
+| Dashboard | ⚠️&nbsp;Restricted | ⚠️&nbsp;Restricted | ✅ Yes   | For members, scope is limited based on a their teams and the project access granted to those teams. |
+| Projects  | ⚠️&nbsp;Restricted | ⚠️&nbsp;Restricted | ✅ Yes   | Projects assigned to teams are visible to members and managers of those teams. Admins can see all projects. |
+| Findings  | ⚠️&nbsp;Restricted | ⚠️&nbsp;Restricted | ✅ Yes   | Members can perform all triage operations on Projects assigned to them. |
+| Policies  | ❌ No         | ❌ No         | ✅ Yes   |       |
+| Editor    | 👁️&nbsp;Read-only  | 👁️&nbsp;Read-only  | ✅ Yes   | Members can view all rules of an organization, but can't edit or create their own. They can create their own rules in their personal account.      |
+| Settings  | ❌ No         | ⚠️&nbsp;Restricted | ✅ Yes   |       |
```

Comment:
Probably need to add something here about what managers can do on the settings page. I believe they can see the Account tab and the Access tab. On the Access tab they can only see the Members and Teams tabs. I'm not sure what teams and users they can see. I think they can see only the teams they manage but they can see all users?

Reply:
Just kidding - they can see all teams but only edit theirs. And they can see all users but not all projects

Reply:
@sabrinabrogren iirc they can't see the **Members** tab but can see the **Teams** tab, is that right?

Reply:
Sorry, you already answered it:
>  On the Access tab they can only see the Members and Teams tabs.